src/main/java/io/th0rgal/oraxen/mechanics/provided/misc/food/FoodMechanic.java

```diff
@@ -0,0 +1,25 @@
+package io.th0rgal.oraxen.mechanics.provided.misc.food;
+
+import io.th0rgal.oraxen.mechanics.Mechanic;
+import io.th0rgal.oraxen.mechanics.MechanicFactory;
+import org.bukkit.configuration.ConfigurationSection;
+
+public class FoodMechanic extends Mechanic {
+
+    public final int hunger;
+    public final int saturation;
```

Comment:
can you here add an option for "replacement"? aka the item it should give if nothing is here.
Should be able to specify oraxen or vanilla item (look at how drop lets you do this).

essentially how when you drink a milk bucket you get milk